egs/tts/valle_v2/README.md

```diff
@@ -0,0 +1,169 @@
+# VALLE
+## Introduction
+This is an unofficial PyTorch implementation of VALL-E, a zero-shot voice cloning model via neural codec language modeling ([paper link](https://arxiv.org/abs/2301.02111)). 
+If trained properly, this model could match the performance specified in the original paper.
+
+## Change notes
+This is a refined version compared to the first version of VALLE in Amphion, we have changed the underlying implementation to Llama
+to provide better model performance, faster training speed, and more readable codes.
+This can be a great tool if you want to learn speech language models and its implementation.
+
+## Installation requirement 
+
+Set up your environemnt as in Amphion README (you'll need a conda environment, and we recommend using Linux). A GPU is recommended if you want to train this model yourself.
+For inferencing our pretrained models, you could generate samples even without a GPU.
+To ensure your transformers library can run the code, we recommend additionally running:
+```bash
+pip install -U transformers==4.41.2
```

Comment:
To specifiy it in the main `env.sh` as well.

Reply:
resolved